nevergrad/optimization/optimizerlib.py

```diff
@@ -2760,3 +2767,6 @@ def __init__(
         base_optimizer: base.OptCls = NGOpt,
     ) -> None:
         super().__init__(_MSR, locals())
+
+
+NGOptRL = SpecialRL
```

Comment:
I'd rather we keep one name only if possible, why have two names for now?
As usual, adding more names means more breaking changes down the roads, so I prefer being careful.